ambry-clustermap/src/main/java/com.github.ambry.clustermap/StaticClusterAgentsFactory.java

```diff
@@ -117,6 +117,11 @@ public void registerPartitionStateChangeListener(StateModelListenerType listener
             PartitionStateChangeListener partitionStateChangeListener) {
           listeners.add(partitionStateChangeListener);
         }
+
+        @Override
+        public ReplicaSyncUpManager getReplicaSyncUpManager() {
+          return null;
```

Comment:
Minor (thinking out load): for optional fields like this that aren't already set, perhaps we should use Optional<>.  don't need to change it now.

Reply:
Fair point, would probably change this in future PR.